lib/Dialect/FIRRTL/Transforms/CreateSiFiveMetadata.cpp

```diff
@@ -41,6 +41,8 @@ class CreateSiFiveMetadataPass
   DenseSet<Operation *> dutModuleSet;
   // The design under test module.
   FModuleOp dutMod;
+  CircuitOp circuitOp;
```

Comment:
I think to be a proper citizen, this should be reset at the end of runOnOperation, like we do with `dutMod` on line 450.